### DISCUSSION TITLE
Student benefits github

### DISCUSSION THREAD

**Discussion by itsSauraj** [community/community#60451] [GitHub Education] [answered]
Unable to verify as a student in a university.

I am a student of University which is listed on GitHub but as GitHub says it needs an official email account provided by my university but when I talked about this to my HOD he said our university doesn't provide such email to students.

**Comment by GhostOf0days** [comment]


**Reply by itsSauraj** [reply, marked as answer]
Please could you please read my issue once again and provide me with suitable answer

**Comment by jabibamman** [comment]
The same thing happened to me, 

Head over to https://support.github.com/?q=student

Then you'll be able to ask with a Virtual Assistant, "Troubleshoot Student Developer Pack"

You don't need an official e-mail account from your univ no worries:)

[EDIT]

Use this link and create a ticket :  [https://support.github.com/contact/education](https://support.github.com/contact/education?tags=hubberfy_education)

**Reply by GhostOf0days** [reply]
This won't actually help. He needs a school email. The bot will just give guidelines for verification. He could try reaching out to Github Support by creating a ticket.

**Reply by jabibamman** [reply]
Got it, so he can conctact them by this link : [https://support.github.com/contact/education](https://support.github.com/contact/education?tags=hubberfy_education), that's what i did last year.

**Comment by samus-aran** [comment]
Hi @CodeWithSaurabhYadav 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
